data/genesis/imap/errorlimit.rb

```diff
@@ -49,8 +49,10 @@
 current.action = [  
   CreateAccount.new(testAccount.name,testAccount.password),
   
+  
   # set the default IMAP error counter
-  ZMLocalconfig.new('-e imap_max_consecutive_error=5'),
+  RunCommandOnMailbox.new('zmlocalconfig -e imap_max_consecutive_error=0'),
```

Comment:
There will be problems with using any localconfig settings on a multinode system with multiple mailbox servers, especially in an HA environment.  

Also, is this localconfig setting appropriate/used with external `zimbra-imapd` servers?  Have to check that.  I seem to recall an earlier Jira ticket about this.